### PR TITLE
[#171536828] App service module

### PR DESCRIPTION
**Pull request by rchessa**

### CODE REVIEW

File: azurerm_app_service/vars.tf [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  })
+
+  default = {
+    kind     = "elastic"
+    sku_tier = "ElasticPremium"
+    sku_size = "EP1"
+  }
+}
+
+variable virtual_network_info {
```
Comment (uolter) [inline azurerm_app_service/vars.tf]: most likely they are working but according to this documentation:
https://learn.hashicorp.com/terraform/azure/variables_az#defining-variables variable name should be in double quote.
```suggestion
variable "virtual_network_info" {
```

File: azurerm_app_service/vars.tf [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+variable "app_settings_secrets" {
+  type = object({
+    key_vault_id = string
+    map          = map(string)
+  })
+}
+
+variable "ip_restriction" {
+  type    = list(any)
+  default = null
```
Comment (uolter) [inline azurerm_app_service/vars.tf]:
```suggestion
  default = []
```

File: azurerm_app_service/vars.tf [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+variable "resource_group_name" {
+  type = string
+}
+
+variable "log_analytics_workspace_id" {
+  type        = string
+  description = "The log_analytics workspace id"
+}
+
+variable "storage_account_info" {
```
Comment (francescopersico) [inline azurerm_app_service/vars.tf]: Not used

Comment (francescopersico) [inline azurerm_app_service/vars.tf]: Not used. Please remove.

**Pull request merged**